config/samples/elasticsearch/elasticsearch.yaml

```diff
@@ -30,6 +29,9 @@ spec:
         #   securityContext:
         #     privileged: true
         #   command: ['sh', '-c', 'sysctl -w vm.max_map_count=262144']
+        ###
```

Comment:
This is simply a helpful note, commented by default, for anyone using Linkerd 2.x or similar that needs to access the automountServiceAccountToken to inject a side-car proxy